Content.Server/Mind/Components/MindComponent.cs

```diff
@@ -137,18 +137,22 @@ public void Examine(FormattedMessage.Builder message, bool inDetailsRange)
                 _entMan.TryGetComponent<MobStateComponent?>(Owner, out var state) &&
                 state.IsDead();
 
-            if (!HasMind)
+            if (dead)
+            {
+                var noSessionText = $"[color=yellow]{Loc.GetString("mind-component-no-mind-and-dead-text", ("ent", Owner))}[/color]";
+                var hasSessionText = $"[color=red]{Loc.GetString("comp-mind-examined-dead", ("ent", Owner))}[/color]";
+
+                message.AddMarkup(Mind?.Session == null ? noSessionText : hasSessionText);
```

Comment:
Make this a regular if statement so that it doesn't have to throw away a Loc.GetString.